tools/buildsys/src/manifest.rs

```diff
@@ -158,6 +164,7 @@ pub(crate) struct BuildPackage {
 pub(crate) struct BuildVariant {
     pub(crate) included_packages: Option<Vec<String>>,
     pub(crate) image_format: Option<ImageFormat>,
+    pub(crate) supported_arches: Option<Vec<SupportedArch>>,
```

Comment:
If we deserialize this into an `Option<HashSet<SupportedArch>>`, it might make the existence test easier.

Reply:
Oh great point!  I always forget about that handy `contains()` method. :)